c31-shadowlyn/react-nutshell-shadowlyn
# login component minus logic

# Description

When a user enters a username and password in `/login`
Then the input fields should be stored into Session Storage

Applies to issue # 7

## Type of change

Please delete options that are not relevant.


# Testing Instructions
1. `git fetch`
2. `git checkout sb-first-branch`
3. run `npm start`
4. add `/login` to end of URL
5. enter in an email and password
6. head to application and session storage in the Developer console.
7. username/password entered should be set in session storage
# Checklist:
